samples/9-Applications.ps1

```diff
@@ -26,6 +26,7 @@ $app3 = New-MgApplication -displayName "ImplicitWebApp" `
         }  
 
 # Create an registration for an ASP.NET Web App
+$scopeId_UserRead = Find-MgGraphPermission User.Read -ExactMatch -PermissionType Delegated | Select-Object -ExpandProperty Id
```

Comment:
@peombwa you'll want to sign off on having the sample changed to use the new command.

Reply:
This looks good! This is a good example of how `Find-MgGraphPermission` can be used to aid in scope discovery.